rudderstack/integrations/destinations/destination_mixpanel.go

```diff
@@ -0,0 +1,203 @@
+package destinations
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	c "github.com/rudderlabs/terraform-provider-rudderstack/rudderstack/configs"
+)
+
+func init() {
+	c.Destinations.Register("mixpanel", c.ConfigMeta{
+		APIType: "MP",
+		Properties: []c.ConfigProperty{
+			c.Simple("token", "token"),
+			c.Simple("apiSecret", "api_secret"),
+			c.Simple("dataResidency", "data_residency"),
+			c.Simple("people", "people"),
+			c.Simple("setAllTraitsByDefault", "set_all_traits_by_default"),
+			c.Simple("consolidatedPageCalls", "consolidated_page_calls"),
+			c.Simple("trackCategorizedPages", "track_categorized_pages"),
+			c.Simple("trackNamedPages", "track_named_pages"),
+			c.Simple("sourceName", "source_name"),
+			c.Simple("crossSubdomainCookie", "cross_subdomain_cookie"),
+			c.Simple("persistence", "persistence"),
+			c.Simple("secureCookie", "secure_cookie"),
+			c.ArrayWithStrings("superProperties", "property", "super_properties"),
+			c.ArrayWithStrings("peopleProperties", "property", "people_properties"),
+			c.ArrayWithStrings("eventIncrements", "property", "event_increments"),
+			c.ArrayWithStrings("propIncrements", "property", "prop_increments"),
+			c.ArrayWithStrings("groupKeySettings", "groupKey", "group_key_settings"),
+			c.Simple("useNativeSDK.web", "use_native_sdk.0.web"),
+			c.ArrayWithStrings("whitelistedEvents", "eventName", "event_filtering.0.whitelist"),
+			c.ArrayWithStrings("blacklistedEvents", "eventName", "event_filtering.0.blacklist"),
```

Comment:
worth checking if these settings have been changed too, as we have changed the verbiage from whitelist/blacklist to allowlist/denylist.

Reply:
same as above